### ISSUE TITLE
Allow selecting multiple tags when filtering predictions on the profile page

### ISSUE THREAD

**Issue by jaysonvirissimo** [bellroy/predictionbook#232]
It might make sense to expose whether to AND or OR via a toggle.

**Comment by Raemon** [comment]
Yeah I've struggled with this when constructing tag-filter interfaces. Having it appear cleanly/easy-to-understand-at-first-glance is tricky.

I would maybe start by building AND tags and worrying about a more complex operation later (When I imagine using this it'd mostly be to combine different types of tags, i.e. I have tags for "#gut" vs "#considered" opinion, and tags for "#lesswrong" team vs "#politics". I'm more likely to want to combine one of the first two with one of the latter two, than care about the combination of the latter.)

Not sure what use-cases you're imagining.
